From 02acffa5a6cf6a1600f709b5a86cc27028a8e44d Mon Sep 17 00:00:00 2001
From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Thu, 30 Jun 2016 17:24:47 +0200
Subject: [PATCH 1/9] qtwebengine: add missing include

Taken from:
https://raw.githubusercontent.com/clemensg/ptxdist/master/patches/qt-everywhere-opensource-src-5.6.1-1/0103-qtwebengine-add-missing-include.patch

Without this building fails new newer glibc with:

.../touch_evdev_debug_buffer.h:37:15: error: field 'device_name_' has
incomplete type 'std::__cxx11::string {aka std::__cxx11::basic_string<char>}'

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 chromium/ui/events/ozone/evdev/touch_evdev_debug_buffer.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chromium/ui/events/ozone/evdev/touch_evdev_debug_buffer.h b/chromium/ui/events/ozone/evdev/touch_evdev_debug_buffer.h
index ccd3a49..b955a5f 100644
--- a/chromium/ui/events/ozone/evdev/touch_evdev_debug_buffer.h
+++ b/chromium/ui/events/ozone/evdev/touch_evdev_debug_buffer.h
@@ -11,6 +11,7 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <string>
 
 #include "ui/events/ozone/evdev/events_ozone_evdev_export.h"
 
-- 
2.7.4

